### PR TITLE
[TRANSLATION] Improve Simplified Chinese (zh-CN) translation

### PR DESCRIPTION
**Pull request by RatinCN**
## Purpose
Improve Simplified Chinese (zh-CN) translation

## Proposed changes
- Add and improve translation
- Fix mistakes

### CODE REVIEW

**Comment by binarymaster** [comment]
Cc @reactos/lang-chinese

**Comment by katahiromz** [comment]
@RatinCN Is this PR ready to go?

File: dll/shellext/fontext/lang/zh-CN.rc [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ */
+
+LANGUAGE LANG_CHINESE, SUBLANG_CHINESE_SIMPLIFIED
+
+STRINGTABLE
+BEGIN
+    IDS_REACTOS_FONTS_FOLDER "ReactOS 字体文件夹"
+END
+
+
```
Comment (katahiromz) [inline dll/shellext/fontext/lang/zh-CN.rc:15]: extra empty line

File: dll/shellext/fontext/lang/zh-CN.rc [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+STRINGTABLE
+BEGIN
+    IDS_COL_FILENAME "文件名"
+    IDS_COL_SIZE "大小"
+    IDS_COL_MODIFIED "修改日期"
+    IDS_COL_ATTR "属性"
+    IDS_COL_ATTR_LETTERS "RHSAC"
+END
+
+
```
Comment (katahiromz) [inline dll/shellext/fontext/lang/zh-CN.rc:30]: extra empty line

File: dll/cpl/hotplug/lang/zh-CN.rc [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    LTEXT "硬件设备(&H)：", IDC_STATIC, 7, 42, 248, 8, WS_CHILD | WS_VISIBLE | WS_GROUP
+    CONTROL "", IDC_SAFE_REMOVE_DEVICE_TREE, "SysTreeView32", TVS_HASBUTTONS | TVS_HASLINES | TVS_DISABLEDRAGDROP | TVS_SHOWSELALWAYS | WS_CHILD | WS_VISIBLE | WS_BORDER | WS_GROUP | WS_TABSTOP, 7, 52, 264, 102
+    LTEXT "", IDC_SAFE_REMOVE_TEXT, 7, 156, 264, 16, WS_CHILD | WS_VISIBLE | WS_GROUP
+    PUSHBUTTON "属性(&P)", IDC_SAFE_REMOVE_PROPERTIES, 158, 178, 55, 14, WS_CHILD | WS_VISIBLE | WS_TABSTOP
+    PUSHBUTTON "停止(&S)", IDC_SAFE_REMOVE_STOP, 216, 178, 55, 14, WS_CHILD | WS_VISIBLE | WS_TABSTOP
+    CONTROL "", IDC_STATIC, "static", SS_CENTER | SS_SUNKEN | WS_CHILD | WS_VISIBLE, 7, 200, 264, 1
+    AUTOCHECKBOX "显示设备组件(&D)", IDC_SAFE_REMOVE_DISPLAY_COMPONENTS, 7, 208, 140, 10, WS_CHILD | WS_VISIBLE | WS_TABSTOP
+    PUSHBUTTON "关闭(&C)", IDCLOSE, 216, 224, 55, 14, WS_CHILD | WS_VISIBLE | WS_TABSTOP
+END
+
```
Comment (katahiromz) [inline dll/cpl/hotplug/lang/zh-CN.rc:26]: extra empty line

File: dll/cpl/hotplug/lang/zh-CN.rc [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+IDM_POPUP_DEVICE_TREE MENU
+BEGIN
+    POPUP ""
+    BEGIN
+        MENUITEM "停止", IDM_STOP
+        MENUITEM "內容", IDM_PROPERTIES
+    END
+END
+
```
Comment (katahiromz) [inline dll/cpl/hotplug/lang/zh-CN.rc:38]: extra empty line

File: dll/cpl/hotplug/lang/zh-CN.rc [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    PUSHBUTTON "属性(&P)", IDC_SAFE_REMOVE_PROPERTIES, 158, 178, 55, 14, WS_CHILD | WS_VISIBLE | WS_TABSTOP
+    PUSHBUTTON "停止(&S)", IDC_SAFE_REMOVE_STOP, 216, 178, 55, 14, WS_CHILD | WS_VISIBLE | WS_TABSTOP
+    CONTROL "", IDC_STATIC, "static", SS_CENTER | SS_SUNKEN | WS_CHILD | WS_VISIBLE, 7, 200, 264, 1
+    AUTOCHECKBOX "显示设备组件(&D)", IDC_SAFE_REMOVE_DISPLAY_COMPONENTS, 7, 208, 140, 10, WS_CHILD | WS_VISIBLE | WS_TABSTOP
+    PUSHBUTTON "关闭(&C)", IDCLOSE, 216, 224, 55, 14, WS_CHILD | WS_VISIBLE | WS_TABSTOP
+END
+
+
+/* Menus */
+
```
Comment (katahiromz) [inline dll/cpl/hotplug/lang/zh-CN.rc:29]: Enough info.
```suggestion
```

File: dll/cpl/hotplug/lang/zh-CN.rc [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+    POPUP ""
+    BEGIN
+        MENUITEM "停止", IDM_STOP
+        MENUITEM "內容", IDM_PROPERTIES
+    END
+END
+
+
+/* Strings */
+
```
Comment (katahiromz) [inline dll/cpl/hotplug/lang/zh-CN.rc:41]: Enough info.
```suggestion
```

File: dll/shellext/fontext/lang/zh-CN.rc (hunk 1 above)
Comment (RatinCN) [reply dll/shellext/fontext/lang/zh-CN.rc]: committed

File: dll/shellext/fontext/lang/zh-CN.rc (hunk 2 above)
Comment (RatinCN) [reply dll/shellext/fontext/lang/zh-CN.rc]: committed

File: dll/cpl/hotplug/lang/zh-CN.rc (hunk 3 above)
Comment (RatinCN) [reply dll/cpl/hotplug/lang/zh-CN.rc:26]: committed

File: dll/cpl/hotplug/lang/zh-CN.rc (hunk 4 above)
Comment (RatinCN) [reply dll/cpl/hotplug/lang/zh-CN.rc:35]: committed

File: dll/cpl/hotplug/lang/zh-CN.rc (hunk 5 above)
Comment (RatinCN) [reply dll/cpl/hotplug/lang/zh-CN.rc]: committed

File: dll/cpl/hotplug/lang/zh-CN.rc (hunk 6 above)
Comment (RatinCN) [reply dll/cpl/hotplug/lang/zh-CN.rc]: committed

**Comment by katahiromz** [comment]
Have a wait for bots.

**Comment by RatinCN** [comment]
Sure

**Pull request merged**